roles/openstack/nova/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+# General defaults
+
+prefix: "{{ provisioner.prefix | default(random.uuid) }}"
+
+# Instances defaults
+
+auto_ip: no
+config_drive: no
+instance_image: centos-minimal
```

Comment:
Can we assume that all users have centos-minimal? Don't think so... so we should use default that suits us the best which isn't centos-minimal imo.